nss-day-cohort-47/reactive-nutshell-the-fantastic-404s
# Js 25 event entry form

# Description
Event entry form.

This is the initial entry f
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions

With React app running and json server
http://localhost:3000/events/create
and the form should be present for the user to see, with interactivity.  
# Checklist:
- [ ] I have added test instructions that prove my fix is effective or that my feature work
